components/board/register/register.go

```diff
@@ -10,4 +10,5 @@ import (
 	_ "go.viam.com/rdk/components/board/numato"
 	_ "go.viam.com/rdk/components/board/pi"
 	_ "go.viam.com/rdk/components/board/ti"
+	_ "go.viam.com/rdk/components/board/customlinux"
```

Comment:
IMPORTANT: remember to undo this line before merging. It's only here so that this can be tested on actual hardware before the PR is done.